tools.yml

```diff
@@ -10,9 +10,21 @@ tools:
     rules: []
     rank: |
       helpers.weighted_random_sampling(candidate_destinations)
+  testtoolshed.g2.bx.psu.edu/repos/jchung/generate_count_matrix/generate_count_matrix/.*:
+    mem: 7.6
+  testtoolshed.g2.bx.psu.edu/repos/simon-gladman/phyloseq_filter/biom_filter/.*:
+    mem: 60.8
+  testtoolshed.g2.bx.psu.edu/repos/simon-gladman/phyloseq_ordination_plot/phyloseq_ordinate/.*:
+    mem: 60.8
+  toolshed.g2.bx.psu.edu/repos/artbio/repenrich/repenrich/.*:
+    cores: 8
+  toolshed.g2.bx.psu.edu/repos/bebatut/cdhit/cd_hit_est/.*:
+    cores: 4
   toolshed.g2.bx.psu.edu/repos/bgruening/alevin/alevin/.*:
     cores: 8
     mem: 92
+  toolshed.g2.bx.psu.edu/repos/bgruening/augustus/augustus/.*:
+    mem: 30.4
```

Comment:
It's not and artifact - GA's config uses a lot of floats since mem = cores * 3.8 a lot of the time. I'll change it to an integer value here for simplicity but I have no general objection to using floats for mem. On GA we are always transforming it to int(mem*1024) to pass to slurm.